ddl/ddl_api.go

```diff
@@ -4238,6 +4284,11 @@ func checkIsDroppableColumn(ctx sessionctx.Context, is infoschema.InfoSchema, sc
 	if err != nil {
 		return false, errors.Trace(err)
 	}
+	// Check the column with TTL config
+	err = checkDropColumnWithTTLConfig(tblInfo, colName.L)
```

Comment:
should we also check it in the ddl worker?

Reply:
Yes. Fixed.